README.md

```diff
@@ -5,8 +5,8 @@ This project was bootstrapped with [Create React App](https://github.com/wmonk/c
 
 The backend is developed separately and their communication is enabled via CORS support on the backend.
 
-Upon Running the code, you will need to edit config/server.json and supply:
-* `url` - URL to the TermIt server (see https://github.com/kbss-cvut/termit for details)
+To run the TermIt UI, it is necessary to provide a value for `REACT_APP_SERVER_URL` representing the URL of the backend to connect to. 
```

Comment:
tohle s tím asi nesouvisí?

Reply:
Ne, nesouvisí. Jen jsem si všimnul, že to je outdated, tak jsem to upravil.